infrastructure/disk.tf

```diff
@@ -92,9 +92,20 @@ resource "aws_s3_bucket" "data_refinery_transcriptome_index_bucket" {
   }
 }
 
+resource "aws_s3_bucket" "data_refinery_qn_target_bucket" {
+  bucket = "data-refinery-s3-qn-target-${var.user}-${var.stage}"
+  acl    = "public-read"
+  force_destroy = "${var.static_bucket_prefix == "dev" ? true : false}"
+
+  tags {
+    Name        = "data-refinery-s3-qn-target-${var.user}-${var.stage}"
+    Environment = "${var.stage}"
+  }
+}
+
 resource "aws_s3_bucket" "data_refinery_compendia_bucket" {
   bucket = "data-refinery-s3-compendia-${var.user}-${var.stage}"
-  acl    = "public-read"
+  acl    = "private"
```

Comment:
Why do compendia results need to be private?

Reply:
Well my thinking was that they contain the data from specific samples, as per this comment: https://github.com/AlexsLemonade/refinebio/issues/731#issuecomment-469443058. However I guess they could be smashed up enough that it's not a risk? @cgreene could you clarify your previous comment to indicate if compendia should be treated more like dataset downloads or more like QN targets/tx-indices?

Reply:
Maybe it's that we're expecting to distribute those via Zenodo?

Reply:
Conferred with @lemonadejay - we should require the checkbox for compendia as well 👍